src/SFML/Window/WindowImpl.hpp

```diff
@@ -128,14 +128,14 @@ class WindowImpl
     /// The \a block parameter controls the behavior of the function
     /// if no event is available: if it is true then the function
     /// doesn't return until a new event is triggered; otherwise it
-    /// returns an empty event to indicate that no event is available.
+    /// returns `std::nullopt` to indicate that no event is available.
     ///
-    /// \param block Use true to block the thread until an event arrives
+    /// \param block Use `true` to block the thread until an event arrives
     ///
-    /// \return The event; can be `Empty` (convertible to `false`) if not blocking
+    /// \return The event; can be `std::nullopt` (convertible to `false`) if not blocking
```

Comment:
```suggestion
    /// \return The event, otherwise `std::nullopt` if not blocking
```

Reply:
This suggested change doesn't seem correct, it's not always `std::nullopt` if non blocking, but it *can be* `std::nullopt` only if it's non blocking